gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -113,6 +119,25 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
     viewModel.initialize(EditObservationFragmentArgs.fromBundle(getArguments()));
   }
 
+  private void fetchResponsesAndErrors() {
+    for (AbstractFieldViewModel fieldViewModel : fieldViewModels) {
+      String fieldId = fieldViewModel.getField().getId();
+      viewModel.onResponseChanged(fieldId, fieldViewModel.getResponse());
+      viewModel.onErrorChanged(fieldId, fieldViewModel.getError().getValue());
+    }
+  }
+
+  @Override
+  public void onPause() {
+    fetchResponsesAndErrors();
```

Comment:
Why do we need to call this in onPause()? Will this cause validation errors to show even if the user hasn't started entering data?